src/encoded/static/components/objectutils.js

```diff
@@ -253,6 +253,14 @@ export function donorDiversity(dataset) {
     return diversity;
 }
 
+/**
+ * Get the control type part of the url
```

Comment:
This comment implies to me that you pass it a query string or URL that possibly contains “control_type!=*”, and it gets the control_type part from it, or maybe its value.